src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --roboto-font: 'Roboto', Arial, sans-serif;
+  --header-size: 60px;
+}
+
+html {
+  font-family: var(--roboto-font);
+  padding: 0;
+  margin: 0;
+}
+
 body {
+  padding: 0;
+  margin: 0;
```

Comment:
It's not recommended to use the '*' selector as it impacts performance. You should set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser. Here, you've used the 'body' and 'html' tags to reset the padding and margin, which is fine, but be cautious about using the '*' selector in other cases.